libs/shared-web/src/hooks/useLocalThemePreference.ts

```diff
@@ -1,9 +1,24 @@
 import { useLocalStorage } from '@mantine/hooks';
 
+const STORAGE_KEY = 'mantine-theme';
+const DEFAULT_THEME_TITLE = 'Match System Appearance';
+
+export enum ColorSchemePreferenceEnum {
+  DARK = 'dark',
+  LIGHT = 'light',
+  SYSTEM = 'system',
+}
+
+export const THEME_TITLE_LOOKUP: Record<ColorSchemePreferenceEnum, string> = {
+  [ColorSchemePreferenceEnum.DARK]: 'Dark Theme',
+  [ColorSchemePreferenceEnum.LIGHT]: 'Light Theme',
+  [ColorSchemePreferenceEnum.SYSTEM]: DEFAULT_THEME_TITLE,
+};
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** Nicely done!‏